src/request.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+// 'use strict';
+
+// const axios = require('axios');
+
+// const BASE = 'http://localhost:8080';
+// const pathname = '/hello/world/123';
+// const search = '?x=1&search=some';
+
+// const href = BASE + pathname + search;
+
+// // eslint-disable-next-line no-console
+// console.log(href);
+
+// axios.get(href)
+//   .catch(() => console.log('error occured'));
```

Comment:
This file must be either removed or be called something like testServer with working code (without commented part)